File: client/src/types/banner.ts
export interface Banner {
  id: string
  imageUrl: string
  title: string
  subtitle: string | null
  linkUrl: string
  displayOrder: number
  isActive: boolean
}

export interface CreateBannerRequest {
  title: string
  subtitle?: string | null
  linkUrl: string
  displayOrder: number
  isActive: boolean
}

export interface UpdateBannerRequest {
  title: string
  subtitle?: string | null
  linkUrl: string
  displayOrder: number
  isActive: boolean
}

File: src/Domain/Entities/Banner.cs
using HungStore.Domain.Common;

namespace HungStore.Domain.Entities;

public class Banner : BaseEntity
{
    public string ImageUrl { get; set; } = string.Empty;
    public string Title { get; set; } = string.Empty;
    public string? Subtitle { get; set; }
    public string LinkUrl { get; set; } = string.Empty;
    public int DisplayOrder { get; set; }
    public bool IsActive { get; set; } = true;
}
